CRM/Utils/Date.php

```diff
@@ -1579,6 +1579,16 @@ public static function relativeToAbsolute($relativeTerm, $unit) {
             $from = self::intervalAdd('second', 1, $from);
             break;
 
+          case 'ending_6':
```

Comment:
this shouldn't be necessary - also check the PR on the ticket for where to add a unit test